packages/apps/transfer/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@kadena/transfer",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "_phase:build": "npm run build",
+    "_phase:test": "npm run test",
+    "build": "next build",
+    "dev": "next dev",
+    "eject": "react-scripts eject",
+    "generate:icons": "npx @svgr/cli --out-dir src/resources/svg/generated -- src/resources/svg",
+    "postinstall": "rushx generate:icons",
+    "lint": "next lint",
+    "lint-staged": "lint-staged",
+    "lint:fix": "eslint --fix .",
+    "start": "next start",
+    "test": ""
```

Comment:
I'd like to have some tests (maybe to add later on). 

Doesn't need to be high coverage, but to test the base features with happy flows. This gives us more certainty when releasing new versions of underlying libs as well